### PR TITLE
Mid : attrd: Ignoring delayed updating of attributes when integrating disjointed clusters without stonith.

### PR DESCRIPTION
**Pull request by HideoYamauchi**
Hi All,

This PR is a modified version of the content discussed in the next PR.
 - https://github.com/ClusterLabs/pacemaker/pull/1625

Currently, this patch works quite as expected, but it is not perfect.
If election of attrd is delayed and problems arise, please set the transition-delay parameter.

Best Regards,
Hideo Yamauchi.

### CODE REVIEW

File: daemons/attrd/attrd_commands.c [hunk 1]
```diff
@@ -871,7 +874,14 @@ attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
...
     } else {
-        crm_trace("Unchanged %s[%s] from %s is %s", attr, host, peer->uname, value);
+        if (is_force_write && a->timeout_ms && a->timer) {
+ 	    /* Save forced writing and set change flag. */
+ 	    /* The actual attribute is written by Writer after election. */
+ 	    crm_trace("Unchanged %s[%s] from %s is %s(Set the forced write flag)", attr, host, peer->uname, value);
+ 	    a->force_write = TRUE;
+ 	} else {
+            crm_trace("Unchanged %s[%s] from %s is %s", attr, host, peer->uname, value);
+        }
```
Comment (kgaillot) [inline daemons/attrd/attrd_commands.c]: There appears to be an indentation issue here, maybe tabs vs spaces?

Comment (HideoYamauchi) [reply daemons/attrd/attrd_commands.c]: Hi Ken,

Sorry...
I fixed the indication.

Many thanks,
Hideo Yamauchi.

**Comment by kgaillot** [comment]
Hi Hideo,

Thanks! I think this helps us get to a reliable synchronization of node attributes.

**Comment by HideoYamauchi** [comment]
Thanks!

**Pull request merged**